### ISSUE TITLE
Improvement idea: A11y labels for propagated onClick, onLongClick, onDoubleClick events

### ISSUE THREAD

**Issue by PattaFeuFeu** [saket/telephoto#166]
# Proposal
It would be nice if it was possible to additionally pass `onClickLabel`, `onLongClickLabel`, `onDoubleClickLabel` for assistive technologies, similar to how [`Modifier.combinedClickable`](https://composables.com/docs/androidx.compose.foundation/foundation/modifiers/combinedClickable) handles it.

## Potential API

```
ZoomableAsyncImage(
  model: Any?,
  contentDescription: String?,
  gestures: EnabledZoomGestures,
  modifier: Modifier = Modifier,
  state: ZoomableImageState = rememberZoomableImageState(rememberZoomableState()),
  imageLoader: ImageLoader = LocalContext.current.imageLoader,
  alpha: Float = DefaultAlpha,
  colorFilter: ColorFilter? = null,
  alignment: Alignment = Alignment.Center,
  contentScale: ContentScale = ContentScale.Fit,
  onClick: ((Offset) -> Unit)? = null,
  onLongClick: ((Offset) -> Unit)? = null,
  onDoubleClick: DoubleClickToZoomListener = DoubleClickToZoomListener.cycle(),
  clipToBounds: Boolean = true,
  contentPadding: PaddingValues = PaddingValues(0.dp),
  
  onClickLabel: String? = null,
  onLongClickLabel: String? = null,
  onDoubleClickLabel: String? = null,
)
```

# Reasoning

`ZoomableAsyncImage` already allows you to pass a `contentDescription` which is read by TalkBack. If additional functionality is added via click listeners, it should be possible to have TalkBack read those out as well to offer a description on what is possible with that view.
e.g.,
* `onClickLabel = "Hide app UI (to focus on photo)"`
* `onLongClickLabel` = "Add photo favourites"

**Comment by saket** [comment]
Love the idea, but the list of parameters in `ZoomableAsyncImage()` is already huge 😣
